harvester/src/main/java/com/github/reflectoring/infiniboard/harvester/source/sourceConfig/ConfigJobManagerJob.java

```diff
@@ -0,0 +1,84 @@
+package com.github.reflectoring.infiniboard.harvester.source.sourceConfig;
+
+import com.github.reflectoring.infiniboard.harvester.scheduling.SchedulingService;
+import com.github.reflectoring.infiniboard.harvester.source.Source.SourceRetrieveJob;
+import com.github.reflectoring.infiniboard.packrat.source.SourceConfig;
+import com.github.reflectoring.infiniboard.packrat.source.SourceConfigRepository;
+import com.github.reflectoring.infiniboard.packrat.source.UrlSource;
+import com.github.reflectoring.infiniboard.packrat.source.UrlSourceRepository;
+import org.quartz.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class ConfigJobManagerJob implements Job {
```

Comment:
please add JavaDoc and describe functional not technical, what this class is used for.